scanpipe/templates/scanpipe/resource_list.html

```diff
@@ -65,12 +66,17 @@
             {% endif %}
             <td>
               <ul>
-                {% for package in resource.discovered_packages.all %}
+                {% for package in resource.discovered_packages.all|slice:":5" %}
                   <li>
                     <a href="{% url 'project_packages' project.slug %}?purl={{ package.package_url }}">{{ package }}</a>
                   </li>
                 {% endfor %}
               </ul>
+              {% if resource.discovered_packages.all.count > 5 %}
+                <a href="{% url 'resource_detail' project.slug resource.path %}#packages" class="button is-small is-fullwidth">
+                    View {{ resource.discovered_packages.all.count|intcomma }} packages
+                </a>
+              {% endif %}
```

Comment:
Using `count` triggers another SQL query. Since we already evaluated the initial QS, using the length defaultfilter would be better.